### DISCUSSION TITLE
Use nvim-notify instead of default toasts

### DISCUSSION THREAD

**Discussion by alexmozaidze** [NeogitOrg/neogit#499] [Polls]
There are a lot of plugins utilizing the power of configurable and fully controllable notifications with the help of [nvim-notify](https://github.com/rcarriga/nvim-notify), and I think it wouldn't be unwise to use it instead of the default toast system.

The dependency would be optional, in that Neogit will detect if [nvim-notify](https://github.com/rcarriga/nvim-notify) exists, and if not, the toast system will be used as a fallback.

**NOTE:** Keep in mind that I am no developer, it's just to see what majority prefers.

**Comment by gegoune** [comment]
Set `disable_builtin_notifications` to `true` and just use `vim.notify` as you like.
